### ISSUE TITLE
builder vmware-iso not able to export ova locally using relative path to existing ovftool

### ISSUE THREAD

**Issue by aksuna** [hashicorp/packer#4832]
FOR FEATURES:

Builder vmware-iso only provides exporting of OVA using VSphere remotely . In my case I had download the ovftool and I want to be able to use the vmware-iso builder calling the ovftool with a relative path and passing the  source_path where .vmx file is  and the destiny_path where it will be generated the ova.

I need that because I want to use "amazon-import" post processor for import the OVA to amazon. Because this postprocessor only take the artifact ova of the builder , and by default vmware-iso dont provide that artifact. So I dont have a way to produce the ova artifact that i need for this postprocessor locally.

I had attend something like this:
```
"builders":[{
			"type":"vmware-iso",
			"version":"8",
			"iso_url":"http://xxxxxx/xxxxxx_latest.iso",
			"iso_checksum":"553ac605f4755c3c6915506078546a4d",
			"iso_checksum_type":"md5",
			"ssh_username":"xxxxx",
			"ssh_password":"xxxxx",
			"ssh_wait_timeout":"120m",
			"shutdown_command":"poweroff",
			"tools_upload_flavor": "linux",
			"guest_os_type":"centos-64",
			"vm_name":"xxxxxxx-6.4",
			"disk_size":"307200",
			"disk_type_id":"0",
			
	}],

"post-processors": [{
	  		"type": "shell-local",
	  		"inline": ["sudo /usr/lib/vmware-ovftool/ovftool output-vmware-iso/xxxxxx.vmx xxxxxx.ova"]
		},
		{
	    	"type": "amazon-import",
	    	"keep_input_artifact": true,
	  		"access_key": "{{user `aws_access_key`}}",
	  		"secret_key": "{{user `aws_secret_key`}}",
	  		"region": "us-east-1",
	  		"ami_name": "ami-xxxxxxx-latest-{{timestamp}}",
	  		"s3_bucket_name": "xxxxx_ovas",
	  		"s3_key_name":"xxxxxxx.ova",
	  		"tags": {
	    		"Description": "packer amazon-import xxxxxxx OVA {{timestamp}}"
	  		}
  	}]
```

**Comment by aksuna** [comment]
017/04/25 12:03:56 machine readable: vmware-iso,artifact []string{"0", "builder-id", "mitchellh.vmware"}
2017/04/25 12:03:56 machine readable: vmware-iso,artifact []string{"0", "id", "VM"}
2017/04/25 12:03:56 machine readable: vmware-iso,artifact []string{"0", "string", "VM files in directory: output-vmware-iso"}
2017/04/25 12:03:56 machine readable: vmware-iso,artifact []string{"0", "files-count", "5"}
2017/04/25 12:03:56 machine readable: vmware-iso,artifact []string{"0", "file", "0", "output-vmware-iso/disk.vmdk"}
2017/04/25 12:03:56 machine readable: vmware-iso,artifact []string{"0", "file", "1", "output-vmware-iso/xxxxxxx-6.4.nvram"}
2017/04/25 12:03:56 machine readable: vmware-iso,artifact []string{"0", "file", "2", "output-xxxxxxx-iso/xxxxxxx-6.4.vmsd"}
2017/04/25 12:03:56 machine readable: vmware-iso,artifact []string{"0", "file", "3", "output-xxxxxxx-iso/xxxxxxx-6.4.vmx"}
2017/04/25 12:03:56 machine readable: vmware-iso,artifact []string{"0", "file", "4", "output-xxxxxxx-iso/xxxxxxx-6.4.vmxf"}
2017/04/25 12:03:56 machine readable: vmware-iso,artifact []string{"0", "end"}

**Comment by mwhooker** [comment]
I think you should be able to give [`"format": "ova"`](https://www.packer.io/docs/builders/vmware-iso.html#format) in the builder to have it output an OVA file.

I'm sorry there's no example of this on the website. Maybe if you can report back when you have it working I can add it to the docs

**Comment by aksuna** [comment]
I had try that , it didnt work when i just want to generate my ova locally and Im not specifying remote-type ="esx5" because in my case I just want to generate the OVA locally  not using remote VSphere client

**Comment by aksuna** [comment]
I had the latest version packer 1.0.0  and i had install ovftool in /usr/lib/vmware-ovftool/ovftool . But just placing the parameter "format":"ova" dont do nothing actually that only works for remote generation using Vsphere client 

> To use a remote VMware vSphere Hypervisor to build your virtual machine, fill in the required remote_* configurations:

remote_type - This must be set to "esx5".

remote_host - The host of the remote machine.

Additionally, there are some optional configurations that you'll likely have to modify as well:

remote_port - The SSH port of the remote machine

remote_datastore - The path to the datastore where the VM will be stored on the ESXi machine.

remote_cache_datastore - The path to the datastore where supporting files will be stored during the build on the remote machine.

remote_cache_directory - The path where the ISO and/or floppy files will be stored during the build on the remote machine. The path is relative to the remote_cache_datastore on the remote machine.

remote_username - The SSH username used to access the remote machine.

remote_password - The SSH password for access to the remote machine.

remote_private_key_file - The SSH key for access to the remote machine.

format (string) - Either "ovf", "ova" or "vmx", this specifies the output format of the exported virtual machine. This defaults to "ovf". Before using this option, you need to install ovftool.

»

**Comment by mwhooker** [comment]
oh you're right, sorry. Let me look at this again

**Comment by aksuna** [comment]
Im stuck on this I wish to use postprocessor amazon-import but i cant because vmware-iso builder is not able to generate ova artifact

**Comment by mwhooker** [comment]
What's happening is that the amazon-import post-processor looks at the artifact produced by the builder to get the OVA to import.

I think you might be able to use the [artifice](https://www.packer.io/docs/post-processors/artifice.html) post-processor to do what you want. It would look something like


```
{
    "post-processors": [
        [
            {
                "inline": [
                    "sudo /usr/lib/vmware-ovftool/ovftool output-vmware-iso/xxxxxx.vmx xxxxxx.ova"
                ],
                "type": "shell-local"
            },
            {
                "files": [
                    "xxxxxx.ova"
                ],
                "type": "artifice"
            },
            {
                "access_key": "{{user `aws_access_key`}}",
                "ami_name": "ami-xxxxxxx-latest-{{timestamp}}",
                "keep_input_artifact": true,
                "region": "us-east-1",
                "s3_bucket_name": "xxxxx_ovas",
                "s3_key_name": "xxxxxxx.ova",
                "secret_key": "{{user `aws_secret_key`}}",
                "tags": {
                    "Description": "packer amazon-import xxxxxxx OVA {{timestamp}}"
                },
                "type": "amazon-import"
            }
        ]
    ]
}
```

**Comment by aksuna** [comment]
I had try without success , Im generating the local ova , looks like "artifice" postprocessor had clear the list of artifacts provided by the builder and detected the ova file that i had provided but when "amazon-import postprocessor" runs dont detect any artifact.

`vmware-iso: Running post-processor: shell-local
==> vmware-iso: Running post-processor: shell-local
2017/04/26 12:20:49 ui: ==> vmware-iso (shell-local): Post processing with local shell script: /tmp/packer-shell647198435
2017/04/26 12:20:49 packer: 2017/04/26 12:20:49 starting local command: chmod +x "/tmp/packer-shell647198435"; PACKER_BUILDER_TYPE='vmware-iso' PACKER_BUILD_NAME='vmware-iso'  "/tmp/packer-shell647198435"
==> vmware-iso (shell-local): Post processing with local shell script: /tmp/packer-shell647198435
2017/04/26 12:20:51 ui:     vmware-iso (shell-local): Opening VMX source: output-vmware-iso/safeconnect-6.4.vmx
    vmware-iso (shell-local): Opening VMX source: output-vmware-iso/safeconnect-6.4.vmx
2017/04/26 12:20:51 ui:     vmware-iso (shell-local): Opening OVA target: safeconnect-6.4.ova
    vmware-iso (shell-local): Opening OVA target: safeconnect-6.4.ova
2017/04/26 12:20:51 ui:     vmware-iso (shell-local): Writing OVA package: safeconnect-6.4.ova
    vmware-iso (shell-local): Writing OVA package: safeconnect-6.4.ova
2017/04/26 12:30:28 ui:     vmware-iso (shell-local): Transfer Completed
    vmware-iso (shell-local): Transfer Completed
2017/04/26 12:30:28 ui:     vmware-iso (shell-local): Completed successfully
    vmware-iso (shell-local): Completed successfully
2017/04/26 12:30:28 Flagging to keep original artifact from post-processor 'shell-local'
2017/04/26 12:30:28 ui: ==> vmware-iso: Running post-processor: artifice
==> vmware-iso: Running post-processor: artifice
2017/04/26 12:30:28 ui: ==> vmware-iso (artifice): Discarding artifact files: output-vmware-iso/disk.vmdk, output-vmware-iso/safeconnect-6.4.nvram, output-vmware-iso/safeconnect-6.4.vmsd, output-vmware-iso/safeconnect-6.4.vmx, output-vmware-iso/safeconnect-6.4.vmxf
2017/04/26 12:30:28 ui: ==> vmware-iso (artifice): Using these artifact files: safeconnect-6.4.ova
==> vmware-iso (artifice): Discarding artifact files: output-vmware-iso/disk.vmdk, output-vmware-iso/safeconnect-6.4.nvram, output-vmware-iso/safeconnect-6.4.vmsd, output-vmware-iso/safeconnect-6.4.vmx, output-vmware-iso/safeconnect-6.4.vmxf
==> vmware-iso (artifice): Using these artifact files: safeconnect-6.4.ova
2017/04/26 12:30:28 ui: ==> vmware-iso: Running post-processor: amazon-import
==> vmware-iso: Running post-processor: amazon-import
2017/04/26 12:30:28 packer: 2017/04/26 12:30:28 Rendered s3_key_name as safeconnect-6.4.ova
2017/04/26 12:30:28 packer: 2017/04/26 12:30:28 Looking for OVA in artifact
2017/04/26 12:30:28 ui error: Build 'vmware-iso' errored: 1 error(s) occurred:

* Post-processor failed: No OVA file found in artifact from builder
2017/04/26 12:30:28 Builds completed. Waiting on interrupt barrier...
2017/04/26 12:30:28 machine readable: error-count []string{"1"}
2017/04/26 12:30:28 ui error: 
==> Some builds didn't complete successfully and had errors:
2017/04/26 12:30:28 machine readable: vmware-iso,error []string{"1 error(s) occurred:\n\n* Post-processor failed: No OVA file found in artifact from builder"}
2017/04/26 12:30:28 ui error: --> vmware-iso: 1 error(s) occurred:

* Post-processor failed: No OVA file found in artifact from builder
2017/04/26 12:30:28 ui: 
==> Builds finished but no artifacts were created.
2017/04/26 12:30:28 waiting for all plugin processes to complete...
Build 'vmware-iso' errored: 1 error(s) occurred:

* Post-processor failed: No OVA file found in artifact from builder

==> Some builds didn't complete successfully and had errors:
--> vmware-iso: 1 error(s) occurred:

* Post-processor failed: No OVA file found in artifact from builder

==> Builds finished but no artifacts were created.
2017/04/26 12:30:28 /usr/local/packer: plugin process exited
2017/04/26 12:30:28 /usr/local/packer: plugin process exited
2017/04/26 12:30:28 /usr/local/packer: plugin process exited
2017/04/26 12:30:28 /usr/local/packer: plugin process exited
2017/04/26 12:30:28 /usr/local/packer: plugin process exited
`

and this is my configuration:
`    "post-processors": [{
                        "type": "shell-local",
                        "inline": ["sudo /usr/lib/vmware-ovftool/ovftool output-vmware-iso/file.vmx file.ova"]
                },
                {
                "files": [
                    "file.ova"
                ],
                "type": "artifice"
                },
                {
                "type": "amazon-import",
                "access_key": "{{user `aws_access_key`}}",
                "secret_key": "{{user `aws_secret_key`}}",
                "region": "us-east-1",
                "ami_name": "ami-file-latest-{{timestamp}}",
                "s3_bucket_name": "xxxxxxova",
                "s3_key_name":"file.ova",
                "tags": {
                        "Description": "packer amazon-import file OVA {{timestamp}}"
                }
                }]
`

**Comment by mwhooker** [comment]
make sure you follow my example exactly. it needs to have two brackets (`"post-processors": [[ ... ]]`) around the post-processors. This is a [sequence definition](https://www.packer.io/docs/templates/post-processors.html#post-processor-definition)

**Comment by aksuna** [comment]
It had import sucessfull, I just notice that had create two AMIs first one with the original id that AMAZON generates and the second one with the name that are provided in the parameter of "amazon_import" postprocessor

017/04/26 13:51:54 packer: 2017/04/26 13:51:54 starting local command: chmod +x "/tmp/packer-shell369658913"; PACKER_BUILDER_TYPE='vmware-iso' PACKER_BUILD_NAME='vmware-iso'  "/tmp/packer-shell369658913"
2017/04/26 13:51:56 ui:     vmware-iso (shell-local): Opening VMX source: output-vmware-iso/safeconnect-6.4.vmx
    vmware-iso (shell-local): Opening VMX source: output-vmware-iso/safeconnect-6.4.vmx
2017/04/26 13:51:56 ui:     vmware-iso (shell-local): Opening OVA target: safeconnect-6.4.ova
    vmware-iso (shell-local): Opening OVA target: safeconnect-6.4.ova
2017/04/26 13:51:56 ui:     vmware-iso (shell-local): Writing OVA package: safeconnect-6.4.ova
    vmware-iso (shell-local): Writing OVA package: safeconnect-6.4.ova
2017/04/26 14:01:41 ui:     vmware-iso (shell-local): Transfer Completed
    vmware-iso (shell-local): Transfer Completed
2017/04/26 14:01:41 ui:     vmware-iso (shell-local): Completed successfully
    vmware-iso (shell-local): Completed successfully
2017/04/26 14:01:41 Flagging to keep original artifact from post-processor 'shell-local'
2017/04/26 14:01:41 ui: ==> vmware-iso: Running post-processor: artifice
==> vmware-iso: Running post-processor: artifice
2017/04/26 14:01:41 ui: ==> vmware-iso (artifice): Using these artifact files: safeconnect-6.4.ova
==> vmware-iso (artifice): Using these artifact files: safeconnect-6.4.ova
2017/04/26 14:01:41 ui: ==> vmware-iso: Running post-processor: amazon-import
==> vmware-iso: Running post-processor: amazon-import
2017/04/26 14:01:41 packer: 2017/04/26 14:01:41 Rendered s3_key_name as safeconnect-6.4.ova
2017/04/26 14:01:41 packer: 2017/04/26 14:01:41 Looking for OVA in artifact
2017/04/26 14:01:41 packer: 2017/04/26 14:01:41 Creating AWS session
2017/04/26 14:01:41 packer: 2017/04/26 14:01:41 Opening file safeconnect-6.4.ova to upload
2017/04/26 14:01:41 ui:     vmware-iso (amazon-import): Uploading safeconnect-6.4.ova to s3://impulse-ova-6-4-4/safeconnect-6.4.ova
    vmware-iso (amazon-import): Uploading safeconnect-6.4.ova to s3://impulse-ova-6-4-4/safeconnect-6.4.ova
2017/04/26 14:03:50 ui:     vmware-iso (amazon-import): Completed upload of safeconnect-6.4.ova to s3://impulse-ova-6-4-4/safeconnect-6.4.ova
    vmware-iso (amazon-import): Completed upload of safeconnect-6.4.ova to s3://impulse-ova-6-4-4/safeconnect-6.4.ova
2017/04/26 14:03:50 packer: 2017/04/26 14:03:50 Calling EC2 to import from s3://impulse-ova-6-4-4/safeconnect-6.4.ova
2017/04/26 14:03:51 ui:     vmware-iso (amazon-import): Started import of s3://impulse-ova-6-4-4/safeconnect-6.4.ova, task id import-ami-fh1ncn5d
2017/04/26 14:03:51 ui:     vmware-iso (amazon-import): Waiting for task import-ami-fh1ncn5d to complete (may take a while)
    vmware-iso (amazon-import): Started import of s3://impulse-ova-6-4-4/safeconnect-6.4.ova, task id import-ami-fh1ncn5d
2017/04/26 14:03:51 packer: 2017/04/26 14:03:51 Waiting for state to become: completed
2017/04/26 14:03:51 packer: 2017/04/26 14:03:51 Using 2s as polling delay (change with AWS_POLL_DELAY_SECONDS)
2017/04/26 14:03:51 packer: 2017/04/26 14:03:51 Allowing 300s to complete (change with AWS_TIMEOUT_SECONDS)
    vmware-iso (amazon-import): Waiting for task import-ami-fh1ncn5d to complete (may take a while)
    vmware-iso (amazon-import): Import task import-ami-fh1ncn5d complete
2017/04/26 14:57:21 ui:     vmware-iso (amazon-import): Import task import-ami-fh1ncn5d complete
2017/04/26 14:57:21 ui:     vmware-iso (amazon-import): Starting rename of AMI (ami-646a0f72)
    vmware-iso (amazon-import): Starting rename of AMI (ami-646a0f72)
2017/04/26 14:57:21 ui:     vmware-iso (amazon-import): Waiting for AMI rename to complete (may take a while)
    vmware-iso (amazon-import): Waiting for AMI rename to complete (may take a while)
2017/04/26 14:57:21 packer: 2017/04/26 14:57:21 Waiting for state to become: available
2017/04/26 14:57:21 packer: 2017/04/26 14:57:21 Using 2s as polling delay (change with AWS_POLL_DELAY_SECONDS)
2017/04/26 14:57:21 packer: 2017/04/26 14:57:21 Allowing 300s to complete (change with AWS_TIMEOUT_SECONDS)
2017/04/26 15:02:26 ui:     vmware-iso (amazon-import): AMI rename completed
    vmware-iso (amazon-import): AMI rename completed
2017/04/26 15:02:26 packer: 2017/04/26 15:02:26 Repacking tags into AWS format
2017/04/26 15:02:26 ui:     vmware-iso (amazon-import): Adding tag "Description": "packer amazon-import Safeconnect OVA 1493227619"
    vmware-iso (amazon-import): Adding tag "Description": "packer amazon-import Safeconnect OVA 1493227619"
2017/04/26 15:02:26 packer: 2017/04/26 15:02:26 Getting details of ami-1f771209
2017/04/26 15:02:26 packer: 2017/04/26 15:02:26 Walking block device mappings for ami-1f771209 to find snapshots
2017/04/26 15:02:26 ui:     vmware-iso (amazon-import): Tagging snapshot snap-04973552a14e13c52
    vmware-iso (amazon-import): Tagging snapshot snap-04973552a14e13c52
2017/04/26 15:02:26 ui:     vmware-iso (amazon-import): Tagging AMI ami-1f771209
    vmware-iso (amazon-import): Tagging AMI ami-1f771209
2017/04/26 15:02:26 packer: 2017/04/26 15:02:26 Adding created AMI ID ami-1f771209 in region us-east-1 to output artifacts
2017/04/26 15:02:26 ui:     vmware-iso (amazon-import): Deleting import source s3://impulse-ova-6-4-4/safeconnect-6.4.ova
    vmware-iso (amazon-import): Deleting import source s3://impulse-ova-6-4-4/safeconnect-6.4.ova
2017/04/26 15:02:27 ui: Build 'vmware-iso' finished.
2017/04/26 15:02:27 Builds completed. Waiting on interrupt barrier...
2017/04/26 15:02:27 ui: 
==> Builds finished. The artifacts of successful builds are:
2017/04/26 15:02:27 machine readable: vmware-iso,artifact-count []string{"4"}
Build 'vmware-iso' finished.

==> Builds finished. The artifacts of successful builds are:
2017/04/26 15:02:27 machine readable: vmware-iso,artifact []string{"0", "builder-id", "mitchellh.vmware"}
2017/04/26 15:02:27 machine readable: vmware-iso,artifact []string{"0", "id", "VM"}
2017/04/26 15:02:27 machine readable: vmware-iso,artifact []string{"0", "string", "VM files in directory: output-vmware-iso"}
2017/04/26 15:02:27 machine readable: vmware-iso,artifact []string{"0", "files-count", "5"}
2017/04/26 15:02:27 machine readable: vmware-iso,artifact []string{"0", "file", "0", "output-vmware-iso/disk.vmdk"}
2017/04/26 15:02:27 machine readable: vmware-iso,artifact []string{"0", "file", "1", "output-vmware-iso/safeconnect-6.4.nvram"}
2017/04/26 15:02:27 machine readable: vmware-iso,artifact []string{"0", "file", "2", "output-vmware-iso/safeconnect-6.4.vmsd"}
2017/04/26 15:02:27 machine readable: vmware-iso,artifact []string{"0", "file", "3", "output-vmware-iso/safeconnect-6.4.vmx"}
2017/04/26 15:02:27 machine readable: vmware-iso,artifact []string{"0", "file", "4", "output-vmware-iso/safeconnect-6.4.vmxf"}
2017/04/26 15:02:27 machine readable: vmware-iso,artifact []string{"0", "end"}
2017/04/26 15:02:27 ui: --> vmware-iso: VM files in directory: output-vmware-iso
--> vmware-iso: VM files in directory: output-vmware-iso
2017/04/26 15:02:27 machine readable: vmware-iso,artifact []string{"1", "builder-id", ""}
2017/04/26 15:02:27 machine readable: vmware-iso,artifact []string{"1", "id", ""}
2017/04/26 15:02:27 machine readable: vmware-iso,artifact []string{"1", "string", ""}
2017/04/26 15:02:27 machine readable: vmware-iso,artifact []string{"1", "files-count", "0"}
2017/04/26 15:02:27 machine readable: vmware-iso,artifact []string{"1", "end"}
2017/04/26 15:02:27 ui: --> vmware-iso: 
--> vmware-iso: 
2017/04/26 15:02:27 machine readable: vmware-iso,artifact []string{"2", "builder-id", "packer.post-processor.artifice"}
2017/04/26 15:02:27 machine readable: vmware-iso,artifact []string{"2", "id", ""}
2017/04/26 15:02:27 machine readable: vmware-iso,artifact []string{"2", "string", "Created artifact from files: safeconnect-6.4.ova"}
2017/04/26 15:02:27 machine readable: vmware-iso,artifact []string{"2", "files-count", "1"}
2017/04/26 15:02:27 machine readable: vmware-iso,artifact []string{"2", "file", "0", "safeconnect-6.4.ova"}
2017/04/26 15:02:27 machine readable: vmware-iso,artifact []string{"2", "end"}
2017/04/26 15:02:27 ui: --> vmware-iso: Created artifact from files: safeconnect-6.4.ova
--> vmware-iso: Created artifact from files: safeconnect-6.4.ova
2017/04/26 15:02:27 machine readable: vmware-iso,artifact []string{"3", "builder-id", "packer.post-processor.amazon-import"}
2017/04/26 15:02:27 machine readable: vmware-iso,artifact []string{"3", "id", "us-east-1:ami-1f771209"}
2017/04/26 15:02:27 machine readable: vmware-iso,artifact []string{"3", "string", "AMIs were created:\n\nus-east-1: ami-1f771209"}
2017/04/26 15:02:27 machine readable: vmware-iso,artifact []string{"3", "files-count", "0"}
2017/04/26 15:02:27 machine readable: vmware-iso,artifact []string{"3", "end"}
2017/04/26 15:02:27 ui: --> vmware-iso: AMIs were created:

us-east-1: ami-1f771209
2017/04/26 15:02:27 waiting for all plugin processes to complete...
--> vmware-iso: AMIs were created:

us-east-1: ami-1f771209
2017/04/26 15:02:27 /usr/local/packer: plugin process exited
2017/04/26 15:02:27 /usr/local/packer: plugin process exited
2017/04/26 15:02:27 /usr/local/packer: plugin process exited
2017/04/26 15:02:27 /usr/local/packer: plugin process exited
2017/04/26 15:02:27 /usr/local/packer: plugin process exited

**Comment by aksuna** [comment]
I had reviewed again and had clean the previous AMI , keeping the one that is renamed so everything is fine thank you so much :)

**Comment by mwhooker** [comment]
Yay glad it works for you

**Comment by lmayorga1980** [comment]
@mwhooker 

I have the same issue with the following template and with packer `1.1.3`

```json
 "post-processors" : [
     [
     {
      "type": "shell-local",
      "inline": [ "/usr/bin/ovftool packer-win2016gui-vmware/packer-vmware-iso.vmx packer-win2016gui-vmware/packer-vmware-iso.ova" ]
     },
     {
         "files": [
           "packer-vmware-iso.ova"
         ],
         "type": "artifice"
     },
     {
       "type": "amazon-import",
       "access_key": "",
       "secret_key": "",
       "region": "{{user `aws_region`}}",
       "s3_bucket_name": "{{user `aws_s3_bucket_name`}}",
       "s3_key_name": "packer-vmware-iso.ova",
       "keep_input_artifact": true,
       "license_type": "BYOL",
       "tags": {
         "Description": "packer amazon-import {{timestamp}}"
       }
     }
     ]
  ]
```

```shell
[Step 6/8] 2018/01/03 11:13:48 packer: 2018/01/03 11:13:48 Looking for OVA in artifact
[11:13:54][Step 6/8] 2018/01/03 11:13:48 [INFO] (telemetry) ending amazon-import
[11:13:54][Step 6/8] Build 'vmware-iso' errored: 1 error(s) occurred:
[11:13:54][Step 6/8] 2018/01/03 11:13:48 [INFO] (telemetry) found error: No OVA file found in artifact from builder
[11:13:54][Step 6/8] 
[11:13:54][Step 6/8] 2018/01/03 11:13:48 ui error: Build 'vmware-iso' errored: 1 error(s) occurred:
[11:13:54][Step 6/8] * Post-processor failed: No OVA file found in artifact from builder
[11:13:54][Step 6/8] 
[11:13:54][Step 6/8] 
[11:13:54][Step 6/8] ==> Some builds didn't complete successfully and had errors:
[11:13:54][Step 6/8] --> vmware-iso: 1 error(s) occurred:
[11:13:54][Step 6/8] 
[11:13:54][Step 6/8] * Post-processor failed: No OVA file found in artifact from builder
[11:13:54][Step 6/8] 
[11:13:54][Step 6/8] ==> Builds finished but no artifacts were created.
[11:13:54][Step 6/8] * Post-processor failed: No OVA file found in artifact from builder
[11:13:54][Step 6/8] 2018/01/03 11:13:48 Builds completed. Waiting on interrupt barrier...
[11:13:54][Step 6/8] 2018/01/03 11:13:48 machine readable: error-count []string{"1"}
[11:13:54][Step 6/8] 2018/01/03 11:13:48 ui error: 
[11:13:54][Step 6/8] ==> Some builds didn't complete successfully and had errors:
[11:13:54][Step 6/8] 2018/01/03 11:13:48 machine readable: vmware-iso,error []string{"1 error(s) occurred:\n\n* Post-processor failed: No OVA file found in artifact from builder"}
[11:13:54][Step 6/8] 2018/01/03 11:13:48 ui error: --> vmware-iso: 1 error(s) occurred:
[11:13:54][Step 6/8] 
[11:13:54][Step 6/8] * Post-processor failed: No OVA file found in artifact from builder
[11:13:54][Step 6/8] 2018/01/03 11:13:48 ui: 
[11:13:54][Step 6/8] ==> Builds finished but no artifacts were created.
[11:13:54][Step 6/8] 2018/01/03 11:13:48 [INFO] (telemetry) Finalizing.
[11:13:54][Step 6/8] 2018/01/03 11:13:48 waiting for all plugin processes to complete...
[11:13:54][Step 6/8] 2018/01/03 11:13:48 /usr/local/bin/packer: plugin process exited
[11:13:54][Step 6/8] 2018/01/03 11:13:48 /usr/local/bin/packer: plugin process exited
[11:13:54][Step 6/8] 2018/01/03 11:13:48 /usr/local/bin/packer: plugin process exited
[11:13:54][Step 6/8] 2018/01/03 11:13:48 /usr/local/bin/packer: plugin process exited
[11:13:54][Step 6/8] 2018/01/03 11:13:48 /usr/local/bin/packer: plugin process exited
[11:13:54][Step 6/8] 2018/01/03 11:13:48 /usr/local/bin/packer: plugin process exited
[11:13:54][Step 6/8] 2018/01/03 11:13:48 /usr/local/bin/packer: plugin process exited
[11:13:54][Step 6/8] 2018/01/03 11:13:48 [ERR] Error decoding response stream 18: EOF
[11:13:54][Step 6/8] 2018/01/03 11:13:48 /usr/local/bin/packer: plugin process exited
[11:13:54][Step 6/8] 2018/01/03 11:13:48 /usr/local/bin/packer: plugin process exited
[11:13:55][Step 6/8] Process exited with code 1
```

**Comment by SwampDragons** [comment]
@lmayorga1980 this ticket is months old -- would you mind opening a new ticket with your issue and steps to reproduce so we can track it separately?

**Comment by lmayorga1980** [comment]
I think the issue was that the `artifice` should be pointing to the relative path of the file and that includes the directory where the `ova` file resides. All is good now but I think the documentation needs to be clear about it. Do you accept PR's on the docs?

**Comment by SwampDragons** [comment]
I _adore_ PRs on the docs!

**Comment by lmayorga1980** [comment]
done https://github.com/hashicorp/packer/pull/5765/files :+1:
